### PR TITLE
Slab demo

### PR DESCRIPTION
**Pull request by brytul**
Added the build.sh script to this branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# l2switch
+This branch is a mirror of the OpenDaylight l2switch gerrit project
```
Comment (slab14) [inline README.md]: Can we add a comment that it is setup to use the Aluminum release of OpenDayLight. And a note that this removes a dependency on leveldbjni, allowing for the controller to run on ARM32 architectures (e.g., Raspberry Pi).

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+setupJava(){
+    export JAVA_HOME=`type -p javac|xargs readlink -f|xargs dirname|xargs dirname|sed 's/8/11/'`
+    export PATH=$PATH:$JAVA_HOME/bin/
+    export M2_HOME=/usr/share/maven/
+    export M2=$M2_HOME
+    export MAVEN_OPTS='-Xmx1048m -XX:MaxPermSize=512m -Xms256m'
```
Comment (slab14) [inline build.sh]: I think that one of these options (MaxPermSize) is no longer supported in Java 11. I believe that is one of the warnings generated initially. maybe remove -XX:MaxPermSize=512m

File: README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+**Purposes:** Branch is intended to be tested with x86 architectuce enivronments like CloudLab VMs used in our [demo](https://github.com/slab14/IoT_Sec_Gateway).
+
+**How it works:**
+
```
Comment (slab14) [inline README.md:7]: Maybe a little description here?
- build with build.sh
- run with startODL.sh
- SDN controller, assumes a virtual switch is running locally named br0, listening on port 6677. Additionally it assumes that docker is running and listening on port 4243. 

Also add a note that it assumes the use of my branch of ovs. for the ovs-docker-remote command.

**Review by slab14 — CHANGES_REQUESTED** [review]
A couple minor comments.

File: build.sh (hunk 2 above)
Comment (brytul) [reply build.sh]: https://github.com/archimatetool/archi/issues/153

you're talking about this?  I removed it.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+ ![image](https://i.ibb.co/44GHvNQ/image.png)
+ 
+ 
+**1) General setup**
+  - Run the following command to copy the Github repository to your __dataplane nodes__: `git clone https://github.com/brytul/l2switch.git`
+  - On your dataplane node, `cd` into the repository and run **build.sh** and **startODL.sh**.
+    - __Note__: SDN controller assumes a virtual switch is running locally named br0 listening on port 6667.  This build also assumes Docker is running and listening on port 4243. You will need to use [our branch](https://github.com/slab14/ovs/tree/slab) of OVS in order to utilize ovs-docker-remote commands.
```
Comment (slab14) [inline README.md]: br0 listens on port 6677

**Review by slab14 — COMMENTED** [review]
change br0 port to 6677

**Comment by slab14** [comment]
Adding readme, describing operation and dependencies.

**Pull request merged**